chemprop/cli/predict.py

```diff
@@ -219,13 +261,14 @@ def prepare_data_loader(
 def make_prediction_for_models(
     args: Namespace, model_paths: Iterator[Path], multicomponent: bool, output_path: Path
 ):
-    model = load_model(model_paths[0], multicomponent)
-    output_columns = load_output_columns(model_paths[0])
+    mol_atom_bond = False
+    models = [load_model(model_path, multicomponent) for model_path in model_paths]
     bounded = any(
-        isinstance(model.criterion, LossFunctionRegistry[loss_function])
+        isinstance(models[0].criterion, LossFunctionRegistry[loss_function])
         for loss_function in LossFunctionRegistry.keys()
         if "bounded" in loss_function
     )
```

Comment:
Is it possible to check if the `model.criterion` is a class inherited from `BoundedMixin` to decide whether the `bounded` is `True`?